packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -184,8 +184,8 @@ onMounted(() => {
 
 preferences.update('autoScrollingEnabled', props.gql.localSettings.preferences.autoScrollingEnabled ?? true)
 preferences.update('isSpecsListOpen', props.gql.localSettings.preferences.isSpecsListOpen ?? true)
-preferences.update('reporterWidth', reporterWidth.value)
-preferences.update('specListWidth', specListWidth.value)
+preferences.update('reporterWidth', props.gql.localSettings.preferences.reporterWidth ?? reporterWidth.value)
```

Comment:
`useRunnerStyle` takes an optional reporterWidth and specsListWidth. Can we pass the gql values in here which would then allow us to remove all other code changes to this file?